data/134/336/392/7/1343363927.geojson

```diff
@@ -28,6 +28,27 @@
     "name:ara_x_preferred":[
         "K\u016bb"
     ],
+    "name:ceb_x_preferred":[
+        "Kub"
+    ],
+    "name:deu_x_preferred":[
+        "KUB"
+    ],
+    "name:ita_x_preferred":[
+        "KUB"
+    ],
+    "name:jpn_x_preferred":[
```

Comment:
Hmm.. I'm guessing any three-character name will result in a bunch of new airport codes..